nodejs/week2/mealSharing/src/backend/app.js

```diff
@@ -0,0 +1,51 @@
+
+
+const express = require("express");
+const app = express();
+
+
+
+const meals = require("./routes/meals.js");
+
+const cheapmeal = require("./routes/cheapMeals");
+
+const largemeals = require("./routes/largemeals");
+const randomMeal = require("./routes/meal");
+
+const reservations = require("./routes/reservations");
+const randomreservations = require("./routes/randomReservation");
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing");
+});
+
+app.get("/meals", (request, response) => {
+  response.send(meals);
+});
+
+app.get("/cheapMeals", (request, response) => {
```

Comment:
`"/cheapMeals"` ➡️  `"/cheap-meals"`

Avoid capital letters in urls, use dashes to split words :)